src/BarrelTOFTracker_geo.cpp

```diff
@@ -0,0 +1,399 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2022 Whitney Armstrong
```

Comment:
```suggestion
// Copyright (C) 2022 - 2024, Whitney Armstrong, Chun Yuen Tsang
```

Reply:
@ssedd1123 This is to acknowledge that your contribution is licensed under LGPL3+.